Frameworks/CoreImage/CIImage.mm

```diff
@@ -211,12 +209,11 @@ + (CIImage*)imageWithData:(NSData*)data {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes options not supported
 */
-+ (CIImage*)imageWithData:(NSData*)data options:(NSDictionary*)d {
-    UNIMPLEMENTED();
-    return StubReturn();
++ (CIImage*)imageWithData:(NSData*)data options:(NSDictionary*)options {
+    return [CIImage imageWithData:data];
```

Comment:
nit: prefer calling the options `init` too, so that when that gets brought online this automatically works (!)